.github/workflows/containerize-agent.yaml

```diff
@@ -55,14 +59,17 @@ jobs:
         id: login-ecr
         uses: aws-actions/amazon-ecr-login@v2
 
-      - name: Build Test Image
+      # https://docs.docker.com/build/ci/github-actions/test-before-push/
+      - name: Build, No Push
         uses: docker/build-push-action@v4
         with:
           context: .
           platforms: linux/amd64
+          push: false
           load: true
           tags: greatexpectations/agent:latest
```

Comment:
Should we tag this as a nightly build?

Reply:
At this point, no. It has no functional differences from `latest`, and we're already retagging a semver